tools/rapids-get-pr-artifact

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Echo path to an artifact for a specific PR. Finds and uses the latest commit on the PR.
+#
+# Positional Arguments:
+#   1) repo name
+#   2) PR number
```

Comment:
I would add an optional 4th argument for the commit hash if folks want to lock it. It would default to the current behavior of finding the latest hash.